src/v2/ion/IonResponseHelper.js

```diff
@@ -152,18 +152,18 @@ const isIonErrorResponse = (response = {}) => {
   return response.version;
 };
 
-const isTerminalError = (res = {}) => {
-  if (!Array.isArray(res.remediations) || res.remediations.length !== 1) {
-    return false;
-  }
-
-  const name = res.remediations[0].name;
+const isTerminalError = (formName) => {
   // TODO: use regex on name?
```

Comment:
I think we can remove this comment - seems like a global list of forms that are **terminal** is probably the safest.